### PR TITLE
Improved optuna's plugin docs

### PR DESCRIPTION
**Pull request by elisim**
## Motivation

Used hydra-optuna plugin, wrote some docs improvements:

* Removed lines about pre-release install 
* fixed broken links, added one comment for @omry to check 
* added docs about how to get the ` optimization_results.yaml` file

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebookresearch/hydra/blob/master/CONTRIBUTING.md)?

Yes

## Test Plan

Only docs 

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

https://github.com/facebookresearch/hydra/issues/1797

BTW, do you think it should be mentioned that the plugin supports numpy<1.20.0 (for now)?

Thank you,
Eli

### CODE REVIEW

File: website/versioned_docs/version-1.1/plugins/optuna_sweeper.md [hunk 1]
```diff
@@ -41,11 +40,11 @@ defaults:
 
 Alternatively, add `hydra/sweeper=optuna` option to your command line.
 
-The default configuration is <GithubLink to="plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/config.py">here</GithubLink>.
+The default configuration is <GithubLink to="plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/config.py">here</GithubLink>. # Omry: broken link
```
Comment (jieru-hu) [inline website/versioned_docs/version-1.1/plugins/optuna_sweeper.md:43]: this is not a broken link (you can try clicking it in our documentation. ) GithubLink is a component.

File: website/versioned_docs/version-1.1/plugins/optuna_sweeper.md [hunk 2]
```diff
@@ -41,11 +40,11 @@ defaults:
...
 
 ## Example 1: Single-Objective Optimization
 
-We include an example in this directory. `example/sphere.py` implements a simple benchmark function to be minimized.
+We include an example in [this directory](https://github.com/facebookresearch/hydra/tree/main/plugins/hydra_optuna_sweeper/example). `example/sphere.py` implements a simple benchmark function to be minimized.
```
Comment (jieru-hu) [inline website/versioned_docs/version-1.1/plugins/optuna_sweeper.md:47]: pls use `GithubLink` here as well.

File: website/versioned_docs/version-1.1/plugins/optuna_sweeper.md [hunk 3]
```diff
@@ -100,6 +99,17 @@ You can also override the search space parametrization:
 python example/sphere.py --multirun 'x=interval(-5.0, 5.0)' 'y=interval(0, 10)'
 ```
 
+You might find the `optimization_results.yaml` file (i.e. best params and best value) under `multirun` logs folder:
+
+```yaml
+name: optuna
+best_params:
+  x: 0.0
+  'y': 0
```
Comment (jieru-hu) [inline website/versioned_docs/version-1.1/plugins/optuna_sweeper.md:108]: is this a typo? why does y have single quotes around it?

**Review by jieru-hu — CHANGES_REQUESTED** [review]
thank you!
we deploy a preview doc with every PR, and you can find yours here (click on deploy/netlify). From the preview you can double check the link and format etc. 
image

**Comment by jieru-hu** [comment]
could you also rebase on master and force push to this branch? (I just merged in a fix for CI)
thanks!

File: website/versioned_docs/version-1.1/plugins/optuna_sweeper.md (hunk 3 above)
Comment (omry) [reply website/versioned_docs/version-1.1/plugins/optuna_sweeper.md:108]: To disambiguate the string 'y'  from the yaml boolean y.

**Review by omry — COMMENTED** [review]
This should also be updated in the primary non-versioned docs.

**Comment by elisim** [comment]
> we deploy a preview doc with every PR, and you can find yours here (click on deploy/netlify). From the preview you can double check the link and format etc.

Great, saw it. Thank you.

> could you also rebase on master and force push to this branch? (I just merged in a fix for CI)

Done. Hope I did it right 

Thanks

File: website/versioned_docs/version-1.1/plugins/optuna_sweeper.md (hunk 2 above)
Comment (jieru-hu) [reply website/versioned_docs/version-1.1/plugins/optuna_sweeper.md:47]: why was this one resolved @omry ? `GithubLink` is applicable here right?

File: website/docs/plugins/optuna_sweeper.md [hunk 4]
```diff
@@ -45,7 +44,7 @@ The default configuration is <GithubLink to="plugins/hydra_optuna_sweeper/hydra_
 
 ## Example 1: Single-Objective Optimization
 
-We include an example in this directory. `example/sphere.py` implements a simple benchmark function to be minimized.
+We include an example in [this directory](https://github.com/facebookresearch/hydra/tree/main/plugins/hydra_optuna_sweeper/example). `example/sphere.py` implements a simple benchmark function to be minimized.
```
Comment (jieru-hu) [inline website/docs/plugins/optuna_sweeper.md:47]: Thanks for adding this. could you update the link in both versions to use GithubLink? that way the link will be directed to the correct branch. Thank you.

**Review by jieru-hu — CHANGES_REQUESTED** [review]
thank you! looks good, one nit.

**Comment by elisim** [comment]
@jieru-hu  Thank you for pointing it out! 

I fixed it. Checked also in deploy/netlify

image

**Comment by jieru-hu** [comment]
Thank you! @elisim

**Pull request merged**